### PR TITLE
Adds new parts

### PR DESCRIPTION
**Pull request by gracecanfield**
https://respiraworks.workast.io/task/0b493f010d0255c5a0c5f38437538dbf
https://respiraworks.workast.io/task/332851ee1549e08037c67683f523e65c

### CODE REVIEW

File: 2_Research_&_Development/Project-Pinch_Valve/README.md [hunk 1]
```diff
@@ -55,22 +55,24 @@ The above files should not be printed with FDM. Instead, a leaktight method like
...
+| 8      | 6in/15cm | McMaster-Carr | 5894K39             |  8.04 / 2 ft | Buy          | [C][8mcmc]         | Continuous-Flex Soft Tygon PVC Tubing, 1/2" ID, 5/8" OD |
+| 9      |        2 | McMaster-Carr | 9579K67             |  12.63 / 20  | Buy          | [C][9mcmc]         | Easy-Install Double Snap-Grip Clamps, 1/2" to 19/32" ID |
+| 10      |        1 | Digikey       | X-NUCLEO-IHM03A1    |        10.64 | *Ask first*  | [K][10key] [M][9mr] | Stepper driver dev board |
+| 11     |        1 | RespiraWorks  | N/A                 |       N/A.   | *Ask first*  | [3D][113d]         | BASE - 3D printed |
+| 12     |        1 | RespiraWorks  | N/A                 |       N/A    | *Ask first*  | [3D][123d]         | ROTOR - 3D printed |
+| 13     | 6in/15cm | McMaster-Carr | 5894K38             |  8.04 / 2 ft | Buy          | [C][13mcmc]        | **alternative to 7**, tubing 3/8" ID, 5/8" OD |
 
 **WARNING: Item 6 may have to be substituted by another adapter, depending on your choice of tubing here (7 or 12) and
```
Comment (martukas) [inline 2_Research_&_Development/Project-Pinch_Valve/README.md:71]: numbers have shifted. This note is wrong.

File: 2_Research_&_Development/Project-Pinch_Valve/README.md [hunk 2]
```diff
@@ -55,22 +55,24 @@ The above files should not be printed with FDM. Instead, a leaktight method like
...
 your choice of pneumatic system internal diameter. Alternative adapters are listed on main pizza page**
 
 [1amzn]:   https://www.amazon.com/dp/B07Z1J8JWH/ref=cm_sw_r_cp_api_i_d.zUEbRBKGSVW
```
Comment (martukas) [inline 2_Research_&_Development/Project-Pinch_Valve/README.md:74]: link might still work if it was cleaner without the /ref... part?

File: 2_Research_&_Development/Project-Pinch_Valve/README.md (hunk 1 above)
Comment (gracecanfield) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: updated.

File: 2_Research_&_Development/Project-Pinch_Valve/README.md (hunk 2 above)
Comment (gracecanfield) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: If you're commenting on a1, you're commenting on stuff that I didn't change to begin with...

Comment (martukas) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: Sorry, just thought it was an opportune moment to take note of this ;)

File: 2_Research_&_Development/Project-Pinch_Valve/README.md [hunk 3]
```diff
@@ -55,22 +55,24 @@ The above files should not be printed with FDM. Instead, a leaktight method like
...
+| 8      | 6in/15cm | McMaster-Carr | 5894K39             |  8.04 / 2 ft | Buy          | [C][8mcmc]         | Continuous-Flex Soft Tygon PVC Tubing, 1/2" ID, 5/8" OD |
+| 9      |        2 | McMaster-Carr | 9579K67             |  12.63 / 20  | Buy          | [C][9mcmc]         | Easy-Install Double Snap-Grip Clamps, 1/2" to 19/32" ID |
+| 10      |        1 | Digikey       | X-NUCLEO-IHM03A1    |        10.64 | *Ask first*  | [K][10key] [M][9mr] | Stepper driver dev board |
+| 11     |        1 | RespiraWorks  | N/A                 |       N/A.   | *Ask first*  | [3D][113d]         | BASE - 3D printed |
+| 12     |        1 | RespiraWorks  | N/A                 |       N/A    | *Ask first*  | [3D][123d]         | ROTOR - 3D printed |
+| 13     | 6in/15cm | McMaster-Carr | 5894K38             |  8.04 / 2 ft | Buy          | [C][13mcmc]        | **alternative to 7**, tubing 3/8" ID, 5/8" OD |
+
+**WARNING: Item 7 may have to be substituted by another adapter, depending on your choice of tubing here (7 or 12) and
```
Comment (martukas) [inline 2_Research_&_Development/Project-Pinch_Valve/README.md:71]: Nope, still wrong. Tubing is 8 or 13

File: 2_Research_&_Development/Project-Pinch_Valve/README.md (hunk 2 above)
Comment (martukas) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: Also, while we are at it, McMaster links should be just part number, without /catalog/ in there, if any still have that. Just to keep an eye out in the future.

File: 2_Research_&_Development/Project-Pinch_Valve/README.md (hunk 3 above)
Comment (gracecanfield) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: Oops - updated.

File: 2_Research_&_Development/Project-Pinch_Valve/README.md (hunk 2 above)
Comment (gracecanfield) [reply 2_Research_&_Development/Project-Pinch_Valve/README.md]: Looks like the page doesn't include any of those (or you updated?) but i'll keep an eye out in the future.

**Pull request merged**